wrappers/dotnet/indy-sdk-dotnet-test/AgentTests/AgentAddIdentityTest.cs

```diff
@@ -1,8 +1,9 @@
-﻿using Indy.Sdk.Dotnet.Wrapper;
+﻿using Hyperledger.Indy.Sdk.AgentApi;
```

Comment:
We discuss with @vimmerru and our vision is `Api` suffix is redundant (here and below: Signus, AnonCreds ...)

Reply:
The issue with this is that it's bad form to have a namespace that conflicts with a class name so an Agent namespace and an Agent class will cause problems, hence the selection of the Api suffix.  The same isn't true of package names in Java.

Reply:
OK, we haven't any another proposal for the suffix, let's use `API`